images/linux/scripts/installers/oras-cli.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+################################################################################
+##  File:  oras-cli.sh
+##  Desc:  Installs ORAS Cli
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+# Determine latest ORAS CLI version
+ORAS_CLI_LATEST_VERSION_URL=https://api.github.com/repos/deislabs/oras/releases/latest
+ORAS_CLI_VERSION=$(curl $ORAS_CLI_LATEST_VERSION_URL | jq '.name' | tr -d '"' | cut -d 'v' -f 2)
```

Comment:
We don't need to parse version:
```
curl -s $ORAS_CLI_LATEST_VERSION_URL | jq -r '.assets[].browser_download_url | select(contains("linux_amd64.tar.
gz"))'

https://github.com/deislabs/oras/releases/download/v0.8.1/oras_0.8.1_linux_amd64.tar.gz
```

Reply:
Thank you, reworked script with your approach.